packages/blade/src/components/Icons/_Svg/Path/__tests__/Path.web.test.tsx

```diff
@@ -25,7 +26,7 @@ describe('<Path />', () => {
 
   it('should not accept dashed props', () => {
     // @ts-expect-error no dashed props are accepted
-    const { container } = renderWithTheme(<Path fill-opacity={0} d="" />);
+    const { container } = renderWithTheme(<Path fill-opacity={0} d="" is="x3d" />);
```

Comment:
How does this resolve the issue?

Reply:
So in the tests, the error indicates that browser does not recognise `path` as a valid dom tag. Using `is="x3d"` in the tests, tells browser to recognise this element as a valid SVG tag.